customfieldnames.civix.php

```diff
@@ -346,3 +346,166 @@ function _customfieldnames_civix_civicrm_alterSettingsFolders(&$metaDataFolders
     $metaDataFolders[] = $settingsDir;
   }
 }
+
+/**
+ * Implements hook_civicrm_buildForm().
+ *
+ * @param string $formName
+ * @param CRM_Core_Form $form
+ */
+function _customfieldnames_civix_civicrm_buildForm($formName, &$form) {
+  $templatePath = realpath(dirname(__FILE__)."/templates");
+  if( ($formName == 'CRM_Custom_Form_Group') && ($form->getAction() == CRM_Core_Action::ADD) ) {
+    $templatePath = realpath(dirname(__FILE__)."/templates");
+    $form->add('text', 'table_name', ts('Set Table Name'), '', FALSE);
+    $form->assign('table_name', TRUE);
+    $form->add('text', 'name', ts('Set Group Name'), '', FALSE);
+    $form->assign('name', TRUE);
+    
+    CRM_Core_Region::instance('page-body')->add(array(
+      'template' => "{$templatePath}/customgroup.tpl"
```

Comment:
move customgroup.tpl to: templates/CRM/LCD/customgroup.tpl
then in this region you can reference it with: 'CRM/LCD/customgroup.tpl'
per the above, the civix helper adds the templates folder as an additional template override path (which can also be used as a template inclusion path). so if we use that structure, we don't have to get the full path name.
I like to use /LCD/ as a subdirectory to ensure we don't have any namespace conflicts with core paths or other extensions.

Reply:
Done